### ISSUE TITLE
Multiple comparison correction in plot_model_comparison

### ISSUE THREAD

**Issue by JasperVanDenBosch** [rsagroup/rsatoolbox#398]
Currently all the MCC happens in the visualization code and not on the `Result` object. Maybe we could do this on the Result object /  consolidate the code. The default options should probably be similar to avoid confusion as to differences between the plot and the `Result` object

**Comment by jdiedrichsen** [comment]
As argued - my strong preference for transparancy is to internally store and use uncorrected p-values. 
For plots and high-level inference routines, MCC should be an option that can be switched on. 

Automatic MCC is confusing and doesn't make sense from a scientific standpoint. 

Joern

**Comment by JasperVanDenBosch** [comment]
Agreed - the default could be False for both - as long as they share the same default behavior or at least make the behavior obvious to prevent this kind of confusion
